p4-16/spec/P4-16-draft-spec.mdk

```diff
@@ -6130,7 +6139,7 @@ table t {
 The ```@name``` annotation directs the compiler to use a different
 local name when generating the external APIs used to manipulate a
 language element from the control plane. It must have a string literal
-argument, which shall not contain the "```.```" character.  In the
+argument.  In the
```

Comment:
When you inline a block into another you have to prepend the name of the callee to all names of the inner instances. If the callee has a relative name, the results are relative paths. I don't see how it can be done otherwise.